docs/specification/draft/client/elicitation.mdx

```diff
@@ -321,12 +526,197 @@ Servers should handle each state appropriately:
+  on behalf of the user, without exposing sensitive credentials or tokens to the
+  MCP client.
+</Note>
+
+#### Understanding the Distinction
+
+- **MCP Authorization**: Required OAuth flow between the MCP client and MCP server (covered in the [authorization specification](../basic/authorization))
+- **Downstream Authorization**: Optional authorization between the MCP server and a third-party resource server, initiated via url elicitation
+
+In downstream authorization, the server acts as both:
+
+- An OAuth resource server (to the MCP client)
+- An OAuth client (to the third-party resource server)
+
+Consider this hypothetical scenario:
+
+- Claude Desktop (the MCP client) connects to Bob's Productivity Server (an HTTPS MCP server)
+- Bob's Productivity Server integrates with Dropbox and Google APIs
+- When Claude calls a tool that requires Dropbox access, the MCP server needs Dropbox credentials
+
+The critical security requirements are:
+
+1. **The Dropbox token MUST NOT transit through Claude Desktop**: The client must never see third-party credentials
+2. **The MCP server MUST NOT use its Claude access token for Dropbox**: That would be [token passthrough](../basic/security_best_practices#token-passthrough), which is forbidden
+3. **The user must authorize the MCP server directly**: The interaction happens outside the MCP protocol
+
+The tokens from these flows are never mixed or passed through. The MCP client's token is for the MCP server only, and the third-party tokens never leave the MCP server.
+
+<Note>
+  For more background, read the [token passthrough
+  section](../basic/security_best_practices#token-passthrough) of the Security
+  Best Practices document to understand why MCP servers cannot act as
+  pass-through proxies.
+</Note>
+
+#### Implementation Pattern
+
+When implementing downstream authorization via url elicitation:
+
+1. The MCP server generates an authorization URL, acting as an OAuth client to the third-party service
+2. The server creates a url elicitation request with this URL
+3. The user completes the OAuth flow directly with the third-party authorization server
+4. The third-party authorization server redirects back to the MCP server
+5. The MCP server securely stores the third-party tokens, bound to the user's identity
+6. Future MCP requests can leverage these stored tokens for API access to the third-party resource server
+
+```mermaid
+sequenceDiagram
+    participant User
+    participant UserAgent as User Agent (Browser)
+    participant 3AS as 3rd Party AS
+    participant 3RS as 3rd Party RS
+    participant Client as MCP Client
+    participant Server as MCP Server
+
+    Client->>Server: tools/call
+    Note over Server: Needs 3rd-party authorization for user
+    Note over Server: Store state (which user this auth flow is for)
+    Server->>Client: ElicitationRequired error<br> (mode: "url", url: "example.com/authorize?...")
```

Comment:
Wouldn't the elicitation happen async to the tool call? IE wouldn't the tool call just be on hold while the elicitation happens?